src/layouts/LandingPage/LandingPage.jsx

```diff
@@ -22,9 +21,30 @@ class LandingPage extends Component {
     this.state = {};
   }
 
+  fetchObjects() {
+    return this.props.getAllObjects();
+  }
+
+  componentDidMount() {
+    // todo: This setTimeout is a temp fix after the refactor to avoid a react loop race condition.
```

Comment:
I will do a follow up PR to fix this temp fix. This fix works fine for now though to fix the react lifecycle loop isssue.